EarlySamples/InitialProject/InitialProject/Assets/Scripts/Test2/UIInputHandler.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using System;
+
+public class UIInputHandler : MonoBehaviour
```

Comment:
Similar to TestPlayInputControl. Why does this need to be a MonoBehaviour?

Reply:
Was a monoBehavior for the Destroy(this) function since they were singletons. However, since making the events of the class static, the singletons have been removed. There is no more need for monoBehavior.